content/news/2019-12-05-mapstruct-and-quarkus.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "MapStruct and Quarkus - a match made in heaven?"
+author: Christian Bandowski
+date: "2019-12-05"
```

Comment:
Looks like you have the same post twice. I commented the other one. Could you have a look at my comments?

Reply:
Oh damn, I renamed the file and mistakenly committed the old one.. I merged the two versions, thanks for the hint!